PopcornTime/Extensions/AppDelegate+Extension.swift

```diff
@@ -21,7 +21,7 @@ extension AppDelegate: PCTPlayerViewControllerDelegate, UIViewControllerTransiti
             } else {
                 let alertController = UIAlertController(title: "No torrents found".localized, message: "Torrents could not be found for the specified media.".localized, preferredStyle: .alert)
                 alertController.addAction(UIAlertAction(title: "OK".localized, style: .default, handler: nil))
-                alertController.show(animated: true)
+                present(alertController, animated:true, completion: nil)
```

Comment:
We din'T have the `completion` block here. Also very minor but not sure about the syntax, space should be added after the `:` as it was before (here and a few other places in the PR).